src/js/components/Settings/SettingsAbout.react.js

```diff
@@ -7,6 +7,10 @@ import AppActions from '../../actions/AppActions';
 
 const shell = electron.shell;
 
+// sth wrong with that, need some check with Webpack
```

Comment:
It's just some images should be real images instead of being bundled in bundle.js